src/styles/blocks/header.scss

```diff
@@ -0,0 +1,49 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__nav-list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__nav-item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__nav-link {
+    font-weight: 500;
+    font-size: 12px;
+    color: var(--color-black);
+    text-decoration: none;
+    text-transform: uppercase;
+    display: block;
+    padding-top: 23px;
+    padding-bottom: 19px;
+
+    &:hover {
+      color: var(--color-blue);
+    }
+  }
+}
+
+.is-active {
+  color: var(--color-blue);
+
+  &::after {
+    content: "";
+    display: block;
+    border-bottom: 4px solid var(--color-blue);
+    border-radius: 8px;
+    position: relative;
+    top: 18.5px;
```

Comment:
```suggestion
    position: absolute;
    bottom: 0;
```

Reply:
Does not work

Reply:
Here is a deeper stuff. 
You need to add to your link `&__nav-link` a rule `position: relative;`

Then your pseudo-element` &::after` will be a little bit different:
Add `position: absolute; bottom: 0; background-color: var(--color-blue); width: 100%; height: 4px; `
Delete `top: 18.5px; border-bottom: 4px solid var(--color-blue);`